FhirToDataLake/deploy/templates/ContainerApp/DeployDicomPipelineToContainerApp.json

```diff
@@ -0,0 +1,354 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "location": {
+        "type": "string",
+        "defaultValue": "East US",
+        "allowedValues": [
+            "Australia East",
+            "Brazil South",
+            "Canada Central",
+            "Central US",
+            "East Asia",
+            "East US",
+            "East US 2",
+            "France Central",
+            "Germany West Central",
+            "Japan East",
+            "Korea Central",
+            "North Central US",
+            "North Europe",
+            "Norway East",
+            "South Africa North",
+            "South Central US",
+            "UK South",
+            "West Europe",
+            "West US",
+            "West US 3"
+        ],
+        "metadata": {
+            "description": "Location for all resources."
+        }
+    },
+    "pipelineName": {
+        "type": "string",
+        "minLength": 3,
+        "maxLength": 24,
+        "metadata": {
+            "description": "The name of the container instance that you wish to create."
+        }
+    },
+    "dicomServerUrl": {
+        "type": "string",
+        "metadata": {
+            "description": "The DICOM server endpoint you wish to get data."
+        }
+    },
+    "dicomApiVersion": {
+        "type": "string",
+        "defaultValue": "V1",
+        "allowedValues": ["V1"]
+    },
+    "serverAuthentication": {
+        "type": "string",
+        "defaultValue": "None",
+        "allowedValues": ["None", "ManagedIdentity"],
+        "metadata": {
+            "description": "Then authentication to access source API server. Set None if you are using an public server API instance."
+        }
+    },
+    "dataStart": {
+        "type": "string",
+        "defaultValue": "1970-01-01 00:00:00 +00:00",
+        "metadata": {
+            "description": "Start timestamp of the data range you want to export."
+        }
+    },
+    "dataEnd": {
+        "type": "string",
+        "defaultValue": "",
+        "metadata": {
+            "description": "End timestamp of the data range you want to export. Will continuous export all data if not specified."
+        }
+    },
```

Comment:
We can remove the start and end timestamp for DICOM.

Reply:
Updated, thanks.